src/Axes.ts

```diff
@@ -401,6 +401,94 @@ export default class Axes extends Component<AxesEvents> {
 		return this;
 	}
 
+	/**
+	 * Stop an animation in progress.
+	 * @ko 재생 중인 애니메이션을 정지한다.
+	 * @method eg.Axes#stop
+	 * @return {eg.Axes} An instance of a module itself <ko>모듈 자신의 인스턴스</ko>
+	 * @example
+	 * const axes = new eg.Axes({
+	 *   "x": {
+	 *      range: [0, 100]
+	 *   },
+	 * });
+	 *
+	 * axes.setTo({"x": 10}, 1000); // start animatation
+	 *
+	 * // after 500 ms
+	 * axes.stop(); // stop animation during movement.
+	 */
+	stop() {
+		this.am.stop(Object.keys(this.axm.get()));
+		return this;
+	}
+
+	/**
+	 * Change the destination of an animation in progress.
+	 * @ko 재생 중인 애니메이션의 목적지를 변경한다.
+	 * @method eg.Axes#updateAnimationPos
+	 * @param {Object.<string, number>} pos The coordinate to move to <ko>이동할 좌표</ko>
+	 * @param {boolean} [restart=false] Whether the animation progress is reset and restarted <ko>애니메이션의 진행도 초기화 및 재시작 여부</ko>
+	 * @return {eg.Axes} An instance of a module itself <ko>모듈 자신의 인스턴스</ko>
+	 * @example
+	 * const axes = new eg.Axes({
+	 *   "x": {
+	 *      range: [0, 200]
+	 *   },
+	 *   "y": {
+	 *      range: [0, 200]
+	 *   }
+	 * });
+	 *
+	 * axes.setTo({"x": 50, "y": 50}, 1000); // trigger animation by setTo
+	 *
+	 * // after 500 ms
+	 * axes.updateAnimationPos({"x": 100, "y": 100}); // animation will end after 500 ms, at {"x": 100, "y": 100}
+	 *
+	 * // after 500 ms
+	 * axes.setTo({"x": 50, "y": 50}, 1000); // trigger animation by setTo
+	 *
+	 * // after 700 ms
+	 * axes.updateAnimationPos({"x": 100, "y": 100}, true); // this works same as axes.setTo({"x": 100, "y": 100}, 300) since restart is true.
+	 */
+	updateAnimationPos(pos: Axis, restart = false) {
```

Comment:
Although duration, pos, and restart can be separated, it seems to be possible to combine them with updateAnimation in the form of an object.

Reply:
Considered about defining a new type for changeable animation param types. Like `ChangeableAnimationParam`, But didn't thought about including `restart` there. 
Maybe using only one parameter for updateAnimation can be better. Also use another name than `ChangeableAnimationParam` if it includes `restart`.